assets/src/dashboard/app/views/myStories/header/index.js

```diff
@@ -114,7 +125,13 @@ function Header({
         handleLayoutSelect={view.toggleStyle}
         currentSort={sort.value}
         pageSortOptions={STORY_SORT_MENU_ITEMS}
-        handleSortChange={sort.set}
+        handleSortChange={useCallback(
```

Comment:
Nit: I'd have put this into a variable, e.g. `const onHandleSortChange = useCallback(...)`

Reply:
i'll move this in my next pr - thanks!